HealthcareTA/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
This is very noisy. Can you boil it down to what you really need and that's different from the one in the repo's root? And if your changes make sense on a global level, just add them to the root.

Reply:
This gitignore file was auto-generated by VS when creating the project, but isn't needed, so I removed it